test/TorchSharpTest/NN.cs

```diff
@@ -1048,6 +1234,23 @@ public void TestTripleMarginWithDistanceLoss()
             }
         }
 
+        [Fact]
+        public void TestTripleMarginWithDistanceLossF()
+        {
+            Func<Tensor, Tensor, Tensor> distance =
+                (x, y) => {
+                    return (x - y).abs();
+                };
+
+            using (Tensor anchor = torch.rand(new long[] { 15, 5 }, requiresGrad: true).neg())
+            using (Tensor positive = torch.randn(new long[] { 15, 5 }, requiresGrad: true))
+            using (Tensor negative = torch.randn(new long[] { 15, 5 })) {
+
+                var result = triplet_margin_with_distance_loss(anchor, positive, negative, distance);
+                Assert.True(true);
```

Comment:
Probably we could check `Assert.Equal(Array.Empty<long>(), result.shape);` or `Assert.False(float.IsNaN(result.item<float>())` to make sure `result` is a float number.

Reply:
Excellent idea.